content/virtualization.md

```diff
@@ -0,0 +1,199 @@
+
+### Selecting the Operating System
+
+![Create a VM](/images/gnome-boxes/boxes-create-1.png)
+
+![Select ISO File](/images/gnome-boxes/boxes-create-2.png)
+
+The first step in the VM creator, is to select the OS installer you would like to use. Most often, these installers will be provided as `.iso` files. To use the ISO file, select "Operating System Image File" at the bottom of the "Select an OS Source" window. Then, a file explorer window will open, and you can navigate to and select your ISO file.
+
+![Identify OS](/images/gnome-boxes/boxes-create-3.png)
+
+After selecting your ISO file, the following window will ask you to identify your operating system. If your operating system is listed in the "Templates" drop down, you can select it. Otherwise, select "Unknown OS" from the list of templates.
+
+### Resource Allocation
+
+![Resource Allocation](/images/gnome-boxes/boxes-create-4.png)
+
+At this point, you should see the Resource Allocation screen where you can select the amount of RAM and the disk size for the VM. The amount of RAM and drive space you give to the VM will be dependant on the hardware requirements for the OS you are looking to install.
+
+If you are unsure what values to use, 8GB of RAM and 60GB for the maximum disk size will be enough for most operating systems to install, with some room for growth as you use the VM.
+
+### Performing the OS Install
+
+![Install the VM](/images/gnome-boxes/boxes-vm-install.png)
+
+After selecting the resources for your VM, the VM will boot the selected operating system installer. Likely, you will see some odd graphical behavior, such as the black bars shown in the above screenshot. This is because some of the drivers needed for the VM are not installed yet. These oddities can be ignored for now, and we will install the drivers later.
+
+The install process for your chosen operating system will be just like any other install, with nothing VM specific at this time. Proceed through the OS install using the OS's guides, or possibly one of the System76 guides linked below:
+
+#### Linux
+
+Install Pop!_OS - [https://support.system76.com/articles/install-pop/](https://support.system76.com/articles/install-pop/)
+
+Install Ubuntu - [https://support.system76.com/articles/install-ubuntu/](https://support.system76.com/articles/install-ubuntu/)
+
+#### Windows
+
+![Windows Express Install](/images/gnome-boxes/boxes-windows-expressinst.png)
+
+When you attempt to create a Windows VM, you will likely see a prompt for the "Express Install" feature of GNOME Boxes. You can certainly use the Express Install with your Windows product key. However, for this article turn off the Express Install, then click "Next" at the top right to move to the standard installer set up.
+
+Install Windows - [https://support.system76.com/articles/windows/](https://support.system76.com/articles/windows/)
```

Comment:
Minor clarity - It may be helpful to give this link some context (like you did for the Linux installers in line 84). It my seem obvious, but may confuse some readers if it's not explicitly stated that the link is offered as an optional reference. Consider this after the last sentence in line 96:

"Reference the below link for standard Windows installation steps."

Reply:
Reworked this section with the Linux installers and Express Install sections.